django_project/frontend/src/store/authSlice.ts

```diff
@@ -71,47 +79,60 @@ export const loginUser = (email: string, password: string) => async (dispatch: A
     axios.defaults.headers['Authorization'] = `Token ${token}`;
 
     dispatch(loginSuccess({ user: response.data.user, token }));
+    dispatch(authSlice.actions.setAuthenticationStatus(true));
```

Comment:
you can remove this dispatch since loginSuccess already set isAuthenticated to True.